tests/functional/test_inference_model_templates.py

```diff
@@ -11,15 +11,15 @@ class TestInferenceModelTemplates(object):
     @pytest.mark.parametrize(
         "model_template, language, env, dataset, target, pos_label, neg_label",
         [
-            (
-                "inference/java_codegen",
-                "java",
-                "java_drop_in_env",
-                "regression_testing_data",
-                "MEDV",
-                None,
-                None,
-            ),
+            # (
```

Comment:
????

Reply:
there is tmp patch sneaked in, for testing